### DISCUSSION TITLE
Icaros 3.3.3

### DISCUSSION THREAD

**Discussion by Xanashi** [Xanashi/Icaros#135] [Releases]
Good Afternoon Everyone,

It's finally time for another stable release. 
I've only made a few changes since the last Beta, here they are:

- Added thumbnailing support for AVIF and HEIC covers in several audio filetypes
- Fixed aspect ratio for some MP4, MOV (and more) files in the MS Photos app
- A few performance improvements 

Thank you all so much for the donations, troubleshooting and help with the translations! ❤️ 
I hope you all enjoy the new release!

<br/>

[![Icaros releases](https://img.shields.io/github/downloads/xanashi/icaros/v3.3.3/total?color=%232B6870&label=DOWNLOADS&style=for-the-badge)](https://github.com/Xanashi/Icaros/releases/tag/v3.3.3)

_______________________________________________________________________________________________________

### Icaros v.3.3.3 Final

- Added support for VVC decoding
- Added support for thumbnailing 'tiled' HEIC files
- Added thumbnailing support of INDD files with an embedded preview
- Added thumbnailing support of EPS files with an embedded preview
- Added thumbnailing support for WEBP, AVIF and HEIC covers in several audio filetypes
- Added AVIF, AI, INDD, EPS, PX, EXR and PDXPRJ to the 'Common' and 'Most Known' presets
- Improved HEIF/AVIF thumbnailing support
- Improved thumbnailing of AI files
- Fixed thumbnailing of several HEVC files
- Fixed colors in EXR thumbnails (changed to sRGB color space)
- Fixed thumbnailing of PXZ files
- Fixed aspect ratio for some MP4, MOV (and more) files in the MS Photos app 
- Fixed MP4 'creation date' property being off by 1 hour
- Fixed some unicode characters showing up wrong in properties
- Fixed rare MKV crash (Thank you Brett! ^__^)
- Fixed issue where Icaros failed to return properties of large files (4GB+) (Thank you dgonyier! ^__^)
- Fixed several UNC Icaros Cache locations issues
- The 'View Size' column on the Locations tab will now only show up in Windows 7 and below, as it is not relevant on newer versions of Windows
- PDF, AI, EPS, INDD, HEIC, HEIF, AVIF, EXR and PSXPRJ are now registered as image formats
- A couple of minor improvements
- Updated GCC 
- Updated FFmpeg

<hr /><em>This discussion was created from the release <a href='https://github.com/Xanashi/Icaros/releases/tag/v3.3.3'>Icaros 3.3.3</a>.</em>

**Comment by Klaus1189** [comment]
Thank you so much ❤️❤️❤️

**Comment by Fredd158** [comment]
Thank you so much. Quick question: My thumbnails are super bright/overexposed, is there a way to fix this?

**Reply by Xanashi** [reply]
Your videos may be HDR encoded, and since Icaros doesn't support HDR, the thumbnails will show up with more dull colors.
See the following issue: #20
